governance.md

```diff
@@ -0,0 +1,393 @@
+and willing to serve in that capacity. The Council will be initially formed
+from the set of existing Core Developers who, as of 2019, have ownership rights
+over the organization.
+
+If a Council member becomes inactive in the project for a period of one year,
+they will be considered for removal from the Council. Before removal, the
+inactive Member will be approached to see if they plan to return
+to active participation. If not, they will be removed immediately upon a Council
+vote. If they plan to return to active participation soon, they will be
+given a grace period of one year. If they don’t return to active participation
+within that time period they will be removed by vote of the Council without
+further grace period. All former Council members can be considered for
+membership again at any time in the future, like any other Project Contributor.
+Retired Council members will be listed on the project website, acknowledging
+the period during which they were active in the Council.
+An inactive BDFL may also be ejected by a consensus of current members under
+these same conditions.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed. This action
+should not be taken lightly. The proposer must provide overwhelming
+justification as to why this action is needed (e.g. the Member violated the
+Code of Conduct) and not other technical reasons (e.g. we have different
+viewpoints on project direction) or personal reasons (e.g. they belong to a
+competitor's organization). We ask that a majority of owners respond for
+quorum, and that those votes come from people employed/associated with at least
+three institutions.
+
+
+### Conflict of interest
+
+It is expected that the Council Members and BDFL will be employed at a wide
+range of companies, universities and non-profit organizations. Because of this,
+it is possible that Members will have conflict of interests. Such conflict of
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment, or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+-   An issue where the person privately gains an advantage from The Project
+    resources, but The Project has no gain or suffers a disadvantage.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused his/herself for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+
+### Voting
```

Comment:
So this section is called "voting" but it doesn't explain anything about voting actually, since voting seems to be eschewed.

Reply:
How do you mean?

Reply:
Any clarification here, @pitrou? As it stands, I'm inclined to leave this as-is.

Reply:
Calling this resolved due to lack of response.  (we're also past the stage where comments like this should slow things down).